cms/src/main/java/uk/nhs/digital/externalstorage/modules/S3ConnectorServiceRegistrationModule.java

```diff
@@ -82,33 +73,25 @@ protected void doShutdown() {
         blockUntilTasksCompleteOrTimeout();
     }
 
-    private void readInConfiguration(final Node moduleConfig) throws RepositoryException {
-        s3Bucket = getValue(AWS_BUCKET, moduleConfig);
-        s3Region = getValue(AWS_REGION, moduleConfig);
-        s3Endpoint = getValue(AWS_S3_ENDPOINT, moduleConfig);
-
-        downloadsMaxConcurrentCount = getValue(DOWNLOAD_MAX_CONC_COUNT, moduleConfig);
-        downloadsShutdownTimeoutInSecs = getValue(DOWNLOAD_SHUTDOWN_TIMEOUT_IN_SECS, moduleConfig);
-
-        uploadsMaxConcurrentCount = getValue(UPLOAD_MAX_CONC_COUNT, moduleConfig);
-        uploadsShutdownTimeoutInSecs = getValue(UPLOAD_SHUTDOWN_TIMEOUT_IN_SECS, moduleConfig);
+    private void readInConfiguration(final Node moduleConfigNode) throws RepositoryException {
+        params = new S3ConnectorServiceRegistrationModuleParams(moduleConfigNode);
     }
 
     private void validateConfiguration() {
 
         try {
-            Validate.notBlank(s3Bucket, "Required argument is missing: " + AWS_BUCKET.getJcrModulePropertyKey());
-            Validate.notBlank(s3Region, "Required argument is missing: " + AWS_REGION.getJcrModulePropertyKey());
+            Validate.notBlank(params.getS3Bucket(), "Required argument is missing: " + S3ConnectorServiceRegistrationModuleParams.KEY_AWS_BUCKET);
```

Comment:
This and subsequent lines that refer to the constants came out quite long (my fault for not having come up with shorter class names 😉).

How about replacing them with one static import, so that the constants can then be referenced without having to qualify them with the class name?:
```java
import static uk.nhs.digital.externalstorage.modules.S3ConnectorServiceRegistrationModuleParams.*;
```